src/Admin/Admin.js

```diff
@@ -20,17 +29,17 @@ class Admin extends React.Component {
             image_url: this.state.image_url,
             number_of_kids: this.state.number_of_kids,
             killed_off: this.state.killed_off,
-            house: this.state.house,
-            description: this.state.description
+            description: this.state.description,
+            house_id: this.state.house_id
         });
 
         this.setState({
             name: '',
             image_url: '',
```

Comment:
you need a handleImage_url cause currently I can't type in that field.

Reply:
Dang surprised I missed that.. Fixed it though. 😁